webserver/templates/user/scraper.js

```diff
@@ -148,7 +148,10 @@ function getLastFMPage(page) {
 
 var version = "1.4";
 var page = 1;
-var numberOfPages = parseInt(document.getElementsByClassName("pages")[0].innerHTML.trim().split(" ")[3]);
+var numberOfPages = 1;
+if (document.querySelector('.pages') != null) {
```

Comment:
do you need to use `===`/`!==` here when comparing to null? I'm not familiar with the difference